projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_base.py

```diff
@@ -510,9 +517,23 @@ def _payload_poster_thread(self):
                         "One or more rows were not ingested.\n"
                         "Exception was: {}".format(str(e))
                     )
+                    exceptions_list.append(e)
             finally:
                 self._payloads_queue.task_done()
 
+            # Complete Post-Ingestion operations
+            try:
+                self._ingester.post_ingest_process(
```

Comment:
In case of ingestion exception, should the exception be a part of the telemetry?

Reply:
Well, all exceptions caught in the ingestion process would be added to the exceptions list, and passed to the post-ingestion process. It would, however, be up to the plugin developer if they want to add the exceptions to the telemetry, or not.